rpc/src/optimistically_confirmed_bank_tracker.rs

```diff
@@ -45,14 +72,23 @@ impl std::fmt::Debug for BankNotification {
                 write!(f, "OptimisticallyConfirmed({slot:?})")
             }
             BankNotification::Frozen(bank) => write!(f, "Frozen({})", bank.slot()),
-            BankNotification::Root(bank) => write!(f, "Root({})", bank.slot()),
+            BankNotification::Root(bank) => write!(f, "Root({})", bank.0.slot()),
         }
     }
 }
 
 pub type BankNotificationReceiver = Receiver<BankNotification>;
 pub type BankNotificationSender = Sender<BankNotification>;
 
+#[derive(Clone)]
+pub struct BankNotificationSenderOption {
```

Comment:
rename to `BankNotificationSenderConfig` since the purpose of this struct is to be a configuration to toggle whether or not to `send_parents` depending if `geyser` is enabled

Reply:
Done